src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/AssessmentService.kt

```diff
@@ -783,6 +793,11 @@ class AssessmentService(
     return AuthorisableActionResult.Success(referralHistoryNoteEntity)
   }
 
+  fun updateAssessmentWithdrawn(assessmentId: UUID) {
```

Comment:
I'm not sure this will save the assessment here, so you'll need to call `assessmentRepository.save` after you set `isWithdrawn`.